explorer/actions.py

```diff
@@ -37,7 +37,10 @@ def _package(queries):
 def _build_zip(queries):
     temp = tempfile.TemporaryFile()
     zip_file = ZipFile(temp, 'w')
-    map(lambda r: zip_file.writestr('%s.csv' % r.title, csv_report(r) or "Error!"), queries)
+    # TODO: Work out why this doesn't work but the latter does
+    # map(lambda r: zip_file.writestr('%s.csv' % r.title, csv_report(r) or "Error!"), queries)
+    for r in queries:
+        zip_file.writestr('%s.csv' % r.title, csv_report(r) or "Error!")
```

Comment:
Yeah, this was too clever anyway. I was in my 'purely functional python' phase.

This fails in python3 because map returns an iterator instead of a list (http://www.diveintopython3.net/porting-code-to-python-3-with-2to3.html#map), so evaluation never happens. It's bad practice to use map() for side-effects anyway, so this is really a solid change. Feel free to remove the comment.